### ISSUE TITLE
Mods all deactivate on Game Launch

### ISSUE THREAD

**Issue by acarperscarper** [mkinfrared/baldurs-gate3-mod-manager#145]
Every time I launch the game from the steam launcher the mod manager deactivates all mods, so I cannot continue in my save at all because it crashes if I launch without the mods enabled. I have auto updates on steam on so it may have updated and broke something, but It feels like a mod manager issue. It lets me restore my previous backups but it doesn't matter becuase they're all deactivated. Any help would be appreciated, because I can't continue anything until it's fixed. My mod list is very long but I don't have anything that should be game breaking as nearly every mod is like cosmetic or ui related.

- Manager Version : 1.5
- Operating System and version : M1 MacOS with Ventura 13.5.1

**Comment by acarperscarper** [comment]
@mkinfrared My whole save file is screwed, Any help please???? Can you at least help me roll back my BG3 to an older version in steam so that I can keep playing from an old update where it worked?? I'm pissed and I'm closed to finishing the game but It crashes on open because all the mods deactivate. :(((((

**Comment by acarperscarper** [comment]
I realized my larian studios folder is in my icloud and my icloud is full, maybe that could be the problem?

**Comment by MarcellaTk** [comment]
Redownload the mods folder in Larian when that happens. All mods are added back but you may have to reactivate them. Happens to me all the time.

**Comment by MarcellaTk** [comment]
> Redownload the mods folder in Larian when that happens. All mods are added back but you may have to reactivate them. Happens to me all the time.

Now I am actually having the same issue despite doing what I usually do. Manager is outdated thats it.

**Comment by BenjaminVanRyseghem** [comment]
usuallly happens when a mod is wrongly installed. Then the game remove the mods list altogether. Try to remove the latest mod you added, and try again maybe?

**Comment by MarcellaTk** [comment]
> usuallly happens when a mod is wrongly installed. Then the game remove the mods list altogether. Try to remove the latest mod you added, and try again maybe?

Maybe that is it? I know I ended up messing with my order so ill test reloading old save orders. If that works then hopefully @acarperscarper is able to fix theirs

**Comment by MarcellaTk** [comment]
> > usuallly happens when a mod is wrongly installed. Then the game remove the mods list altogether. Try to remove the latest mod you added, and try again maybe?
> 
> Maybe that is it? I know I ended up messing with my order so ill test reloading old save orders. If that works then hopefully @acarperscarper is able to fix theirs

YES THAT WAS THE ISSUE! @acarperscarper I suggest re-activating them all one by one. skip any ones you know need to be loaded after another and do those last. Its annoying but as Benjamin said that should fix it

**Comment by BenjaminVanRyseghem** [comment]
glad it helped :smile:
